src/app/domains/categories/domains.transaction-categories.constants.ts

```diff
@@ -1,3 +1,7 @@
+import { TServerEntityId } from 'src/app/common/http/common.http.types';
 import { environment } from 'src/environments/environment';
 
-export const API_TRANSACTION_CATEGORIES_URL = environment.apiUrl + '/categories';
\ No newline at end of file
+export const API_TRANSACTION_CATEGORIES_URL = environment.apiUrl + '/categories';
+
+export const API_TRANSACTION_CATEGORIES_IS_ALREADY_USED_URL = (id: TServerEntityId): string =>
```

Comment:
Where is this endpoint coming from? This should be used https://tea-zagrodnicy.koyeb.app/docs/swagger-ui/index.html#/financial-transaction-category-controller/getFinancialTransactionCategoryById -> there is `financialTransactionsCounter` property